docs/development/index.rst

```diff
@@ -11,7 +11,6 @@ for your own :doc:`custom installation </custom_installs/index>`.
    architecture
    buildenvironments
    design
-   design/index
```

Comment:
I think we should leave these in the TOC, but just exclude them from search. Otherwise they'll be impossible to find.